matomo.php

```diff
@@ -101,6 +101,24 @@ public function onPageInitialized(Event $event)
             }
         }
 
+        // Don't proceed if a blocking cookie is set
+        $blockingCookieName = $config->get('blockingCookie', '');
```

Comment:
this empty string '' is not needed, isnt it? I am not sure, but I think it is not.

Reply:
It is not. Will change.